isogram.json

```diff
@@ -0,0 +1,49 @@
+{
+  "cases": [
+    {
+      "description": "duplicates",
+      "input": "duplicates",
+      "expected": true
+    },
+    {
+      "description": "eleven",
+      "input": "eleven",
```

Comment:
We don't need to have a description if the input is identical... in that case the people writing generators can use the `input` key to create their description.